compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -90,6 +91,9 @@ unsafe fn configure_llvm(sess: &Session) {
         if sess.opts.unstable_opts.print_llvm_passes {
             add("-debug-pass=Structure", false);
         }
+        if llvm_version >= (19, 0, 0) && sess.opts.unstable_opts.lint_llvm_ir {
+            add("-lint-abort-on-error", false);
```

Comment:
I don't think we should hardcode this option. You can add it via `-Cllvm-args=-lint-abort-on-error` if desired.